### PR TITLE
Bug 1353961 Added step on being logged in to upgrade topic

### PR DESCRIPTION
**Pull request by bfallonf**
As per https://bugzilla.redhat.com/show_bug.cgi?id=1353961

I'll ask for an ack in the BZ.

### CODE REVIEW

**Comment by bfallonf** [comment]
Ack given in BZ.

@tpoitras @ahardin-rh @tnguyen-rh @adellape any issues?

File: install_config/upgrading/automated_upgrades.adoc [hunk 1]
```diff
@@ -135,6 +135,12 @@ For any upgrade path, always ensure that you have the latest version of the
 # yum update atomic-openshift-utils
 ----
 
+Lastly, you must be logged in as an admin user for the upgrade to succeed:
```
Comment (tpoitras) [inline install_config/upgrading/automated_upgrades.adoc]: Is "admin" an implied role name here? If so, should it be marked up?

**Comment by bfallonf** [comment]
@tpoitras I get you. Changed to suggestion. Thanks, and I'll merge this.

**Comment by bfallonf** [comment]
[rev_history]
|xref:../install_config/upgrading/automated_upgrades.adoc[Performing Automated Cluster Upgrades]
|Added step about logging in as an administrator.
%

**Pull request merged**